.pre-commit-config.yaml

```diff
@@ -0,0 +1,18 @@
+repos:
+-   repo: git://github.com/pre-commit/pre-commit-hooks
+    sha: v1.1.1
+    hooks:
+    -   id: trailing-whitespace
+        args: ['--no-markdown-linebreak-ext']
+    -   id: check-merge-conflict
+    -   id: debug-statements
+    -   id: check-added-large-files
+    -   id: flake8
+
+-   repo: git://github.com/chewse/pre-commit-mirrors-pydocstyle
+    sha: v2.1.1
+    hooks:
+    -   id: pydocstyle
+        args: ['--select=D204,D201,D209,D210,D212,D300,D403']
+
+exclude: '^(?!bigchaindb/)(?!tests/)'
```

Comment:
I noticed this line in the other PR and just dismissed it as yet another confusing regex. Maybe you could add a comment line above it to say what it _means_. YAML comments begin with `#`, can start anywhere on a line, and continue until the end of the line.

A comment could also be added to help readers understand all those pydocstyle error codes (such as D204), e.g. a link to http://www.pydocstyle.org/en/2.1.1/error_codes.html